lib/SILOptimizer/IPO/DeadFunctionElimination.cpp

```diff
@@ -374,6 +374,7 @@ class DeadFunctionElimination : FunctionLivenessComputation {
         return false;
       });
     }
+    Module->recalculateVTableEntryCache();
```

Comment:
I can't without making the cache public and/or allow public mutability. either that or make the optimization (and any other future optimizations) friends with the Module